### PR TITLE
Remove Clan font usage

### PR DESCRIPTION
**Pull request by chrisdothtml**

### CODE REVIEW

**Comment by AlexMSmithCA** [comment]
Should we be using the new `UberMove` font instead?

File: src/css/normalize.css [hunk 1]
```diff
@@ -200,7 +200,7 @@ html {
 }
 body {
   color: rgba(0, 0, 0, 0.8);
-  font-family: ff-clan-web-pro, '-apple-system', BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+  font-family: '-apple-system', BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```
Comment (nadiia) [inline src/css/normalize.css]: No quotes needed for the `-apple-system` and `sans-serif` I guess

Comment (chrisdothtml) [reply src/css/normalize.css]: Yeah saw your update and fixed :)

**Comment by chrisdothtml** [comment]
!merge

**Pull request merged**